lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -662,15 +666,15 @@ def __init__(self, distribution="norm", shape_parameters=None):
 
     def __repr__(self):
         """Represent the configured plugin instance as a string."""
-        result = ('<FromMeanAndVariance: distribution: {}; '
+        result = ('<ConvertLocationAndScaleParameters: distribution: {}; '
                   'shape_parameters: {}>')
         return result.format(
             self.distribution.name, self.shape_parameters)
 
     def _rescale_shape_parameters(self, mean, std):
         """
-        Rescale the shape parameters for the desired mean and standard
-        deviation for the truncated normal distribution. The shape parameters
+        Rescale the shape parameters for the desired location and scale
+        parameters for the truncated normal distribution. The shape parameters
         for any other distribution will remain unchanged.
 
         For the truncated normal distribution, if the shape parameters are not
```

Comment:
Is this really mean and standard deviation? In the parameters and in the equations

Reply:
I have now updated this function to state that the rescaling uses the location parameter and scale parameter, rather than the mean and standard deviation. This has only renamed variables, without any functional changes, and I think this rescaling still makes sense, in terms of the usage of `location_parameter` and mean e.g. https://stackoverflow.com/questions/36894191/how-to-get-a-normal-distribution-within-a-range-in-numpy/36894937#36894937.